content/guides/container-supported-development/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Faster development and testing with container-supported development"
+linkTitle: Container-supported development
+summary: |
+  Containers don't have to be just for your app. Learn how to run your app's dependent services and other debugging tools to enhance your development environment.
+description: |
+  Use containers in your local development loop to develop and test faster… even if your main app isn't running in containers.
+levels: [beginner]
+params:
+  image: images/learning-paths/container-supported-development.png
+  skill: Beginner
+  time: TBD
+  resource_links: []
+---
+
+Containers provide a standardized ability to build, share, and run applications. While containers are typically used to containerize your application, they also make it incredibly easy to run essential services needed for development. Instead of installing or connecting to a remote database, you can easily launch your own database. But the possibilities don't stop there.
```

Comment:
```suggestion
Containers offer a consistent way to build, share, and run applications across different environments. While containers are typically used to containerize your application, they also make it incredibly easy to run essential services needed for development. Instead of installing or connecting to a remote database, you can easily launch your own database. But the possibilities don't stop there.
```